Updateed README.md with some info about Files UWP (#1853)
files-community/files-uwp: README.md

<img src="Files/Assets/FilesHome.png" width="600px">

## What is Files UWP?
Files UWP is an all in one File Explorer solution. Whether you want to simplify your experience with your files or want to try something new, Files is a one-stop solution to exploring your files on the fly.

## Why should I contribute?

> You should contribute to Files UWP because it helps everyone. When you contribute on GitHub, you are helping the community by fixing bugs that only you might know how to fix. These changes get uploaded to the final product for all consumers to enjoy.

Reviewer: ```suggestion
You should contribute to Files UWP because it helps everyone. When you contribute on GitHub, you are helping the community by fixing bugs that others may not know how to fix. These changes get included in the final product for all users to enjoy.
```
